resources/release/release-announcement-template.md

```diff
@@ -0,0 +1,11 @@
+We're excited to announce the release of OpenSearch ${RELEASE_VERSION}! :tada:! </br>
+
+• Download: https://opensearch.org/downloads.html </br>
+• Release Notes: https://github.com/opensearch-project/opensearch-build/blob/main/release-notes/opensearch-release-notes-${RELEASE_VERSION}.md </br>
+• Documentation: https://opensearch.org/docs/latest/about/ </br>
+For detailed information about what's new, check out our blog post: https://www.opensearch.org/blog/explore-OpenSearch-2-19/ </br>
```

Comment:
I think you missed to change the hardcoded 2.19 value.

Reply:
Sorry had to rebase, merged the old PR and this branch had old commits. Its rebased now.